content/docs/contributing/docs.mdx

```diff
@@ -0,0 +1,274 @@
+---
+title: Contributing to Documentation
+---
+
+## Documentation
+
+As with the other parts of Unikraft, we welcome contributions to the [documentation repository](https://github.com/unikraft/docs).
+Documentation is written using [GitHub Markdown](https://github.github.com/gfm/).
+
+We follow the [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow) for contributions.
+Submit corrections and improvements to documentation as pull requests (PRs) that we will then review and approve in the documentation.
+You can add entire sections if you think a topic is missing or update existing ones with missing information.
+
+You can also open up an issue to signal a problem or a missing part in the documentation that you request someone to fix.
+
+We use English as the documentation language.
+Please proofread your text before submitting a pull request.
+
+Write each sentence on a new line.
+This way, changing one sentence only affects one line in the source code.
+
+Prefer using the 1st person plural.
+Use phrases like "we run the command / app", "we look at the source code", "we provide support".
+
+### Content Layout
+
+The content has the following structure:
+
+```text
+content
+|-- blog/                   # Blog posts, <date>-<blogpost title>.md
+|-- community/              # Community related content (events, general information, etc.)
+|-- docs/                   # Unikraft documentation, split by categories into subdirectories
+|-- guides/                 # Short and on point guides, good for getting started
+|-- hackathons/             # Hackathon landing pages
+|-- imprint.md
+|-- privacy.md
+`-- releases/               # Unikraft releases summaries
+```
```

Comment:
This needs to be updated to the current layout.

Reply:
This is the current layout